src/hotspot/cpu/x86/matcher_x86.hpp

```diff
@@ -191,10 +191,13 @@
     switch(vopc) {
       default:
         return 0;
+      case Op_MulVB:
+        return 7;
```

Comment:
Why do you unconditionally return `7` here? Is it because AVX512 doesn't feature a vector instruction to multiply byte vectors?

Reply:
Yes AFAIK there is no instruction to perform byte vector multiplication directly on x86.